test/drenv/envfile.py

```diff
@@ -1,13 +1,18 @@
 # SPDX-FileCopyrightText: The RamenDR authors
 # SPDX-License-Identifier: Apache-2.0
+import os
 
 import copy
 
 import yaml
 
 
 def load(fileobj, name_prefix=None):
+    if not os.path.isfile(fileobj.name):
+        raise ValueError("Non existing script")
+
     env = yaml.safe_load(fileobj)
```

Comment:
I don't understand this change, and how it fixes the linked issue.

The caller of envfile.load() is responsible to send an open file. This
function does not care if the file exist or is a memory based file (e.g. io.StringIO).

Checking that the file object exists is not a good change, we want to keep
the existing behavior.